nifi-nar-bundles/nifi-asn1-bundle/nifi-asn1-services/src/main/java/org/apache/nifi/jasn1/JASN1Reader.java

```diff
@@ -134,17 +135,32 @@ public class JASN1Reader extends AbstractConfigurableComponent implements Record
         .required(false)
         .build();
 
+    private static final PropertyDescriptor PREPROCESS_OUTPUT_DIRECTORY = new PropertyDescriptor.Builder()
+        .name("additional-preprocesszing-output-directory")
+        .displayName("Additional Preprocessing Output Directory")
+        .description("When set, NiFi will do additional preprocessing steps that creates modified versions of the provided ASN files," +
+                " removing unsupported features in a way that makes them less strict but otherwise should still be compatible with incoming data." +
+                " The original files will remain intact and new ones will be created with the same names in the provided directory." +
+                " For more information about these additional preprocessing steps please see Additional Details - Additional Preprocessing.")
+        .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
```

Comment:
Any thoughts on this? It is still there but in the new property.

Reply:
Although flow-based Variable Registry support will be removed, environment variable-based access can be supported. However, this could also be configured using Parameter Contexts, without the need for environment variables. From that perspective, the current setting might be useful, but not necessary.